### ISSUE TITLE
Error when using from ES module 

### ISSUE THREAD

**Issue by ronnylt** [radix-ui/themes#77]
In a ESM only project, is not possible to import `"@radix-ui/themes"`:

```
(node:94507) Warning: To load an ES module, set "type": "module" in the package.json or use the .mjs extension.
esm-project/node_modules/@radix-ui/themes/dist/esm/index.js:1
export { Theme, useThemeContext, updateThemeAppearanceClass } from './theme';
^^^^^^

SyntaxError: Unexpected token 'export'
    at internalCompileFunction (node:internal/vm:73:18)
    at wrapSafe (node:internal/modules/cjs/loader:1177:20)
    at Module._compile (node:internal/modules/cjs/loader:1221:27)
    at Module._extensions..js (node:internal/modules/cjs/loader:1311:10)
    at Module.load (node:internal/modules/cjs/loader:1115:32)
    at Module._load (node:internal/modules/cjs/loader:962:12)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/translators:165:29)
    at ModuleJob.run (node:internal/modules/esm/module_job:192:25)
    at async DefaultModuleLoader.import (node:internal/modules/esm/loader:246:24)

Node.js v20.3.1
```

```
import { Theme } from "@radix-ui/themes";
         ^^^^^
SyntaxError: Named export 'Theme' not found. The requested module '@radix-ui/themes' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@radix-ui/themes';
const { Theme } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:122:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:188:5)
    at async DefaultModuleLoader.import (node:internal/modules/esm/loader:246:24)
    at async loadESM (node:internal/process/esm_loader:40:7)
    at async handleMainPromise (node:internal/modules/run_main:66:12)

Node.js v20.3.1
```

**Comment by ronnylt** [comment]
https://github.com/radix-ui/themes/issues/51 might be related to this as well, given that some Remix templates use ESM by default.

**Comment by anges244** [comment]
Add this to your remix.config.js: `serverDependenciesToBundle: ["@radix-ui/themes"]` and it should work.

**Comment by ronnylt** [comment]
@anges244 I am not using Remix in this case, is just a simple an ESM native project.

**Comment by MystPi** [comment]
#13 is related as well. Looks like an issue with ESM exports.

**Comment by IgnusG** [comment]
I encountered this issue in vite. What resolved the issue for me was adding radix to the `noExternal` config:

```typescript
	ssr: {
		noExternal: ["@radix-ui/themes"],
	},
``` 

I can't remember where but one of the docs mentioned to [check this website](https://arethetypeswrong.github.io/?p=%40radix-ui%2Fthemes%402.0.0) to see if the issue is with the package and at least according to it it is.

**Comment by jgresham** [comment]
> I encountered this issue in vite. What resolved the issue for me was adding radix to the `noExternal` config:
> 
> ```ts
>   ssr: {
>     noExternal: ["@radix-ui/themes"],
>   },
> ```
> 
> I can't remember where but one of the docs mentioned to [check this website](https://arethetypeswrong.github.io/?p=%40radix-ui%2Fthemes%402.0.0) to see if the issue is with the package and at least according to it it is.

Remix is adding Vite support and the current non-production version has this issue and recommendation and noExternal fix works for it https://remix.run/docs/en/dev/future/vite#esm--cjs

**Comment by vladmoroz** [comment]
This will be fixed in the upcoming 3.0.0 version.

We released `"@radix-ui/themes": "3.0.0-rc.1"` which should work in ESM projects if you need an early version now. Please refer to the changelog for other changes in this release.

**Comment by jgresham** [comment]
> This will be fixed in the upcoming 3.0.0 version.
> 
> We released `"@radix-ui/themes": "3.0.0-rc.2"` which should work in ESM projects if you need an early version now. Please refer to the changelog for other changes in this release.

thank you! Just tried rc2 and it works with my setup! 

cc @itsMapleLeaf
